blog/2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases/index.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "The wasmCloud Postgres capability brings pluggable databases for your apps"
+authors: [ericgregory]
+image: './images/wasmcloud-postgres-capability.webp'
+slug: 2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases
+date: "2024-09-17"
+description: "Learn how to get started with the Postgres capability in wasmCloud."
+---
+
+![The wasmCloud Postgres capability brings pluggable databases for your apps](./images/wasmcloud-postgres-capability.webp)
+
+With wasmCloud, you can build applications out of WebAssembly components and pluggable, reusable [**capabilities**](/docs/concepts/capabilities), so you can focus on the core logic of your application and abstract away the rest, all using open source standards.
+
+If you want to use a database system like **Postgres**, wasmCloud makes it easy to simply plug in the functionality in a reusable, language-agnostic way, without wasting time writing boilerplate code. In this post, we'll show you how to get started with the Postgres capability in wasmCloud.
+
+## Database capabilities speed up development
+
+Enterprise application teams need flexible database options, and for many, [PostgreSQL](https://www.postgresql.org/) is the tool of choice. But like many basic, non-functional requirements, teams can waste a lot of time getting to grips with libraries in this or that language and worrying about how to implement their database for a given project. 
```

Comment:
```suggestion
Enterprise application teams need flexible database options, and for many, [PostgreSQL](https://www.postgresql.org/) is the tool of choice. But like many basic, non-functional requirements, teams waste time configuring client libraries in `$LANGUAGE` and managing connection pools and implementation.
```

I personally like `$LANGUAGE` and the like as placeholders in personal posts - -what do you think? I don't know what our stance on this is